core/templates/dev/head/base.html

```diff
@@ -255,10 +255,20 @@
                           <ul ng-if="!windowIsNarrow" class="nav oppia-navbar-tabs">
                             <create-activity-button></create-activity-button>
 
-                            <li class="oppia-clickable-navbar-element pull-right">
-                              <a class="oppia-navbar-tab" href="/donate" translate="I18N_TOPNAV_DONATE">
+                          <ul ng-if="windowIsNarrow" class="nav oppia-navbar-tabs-narrow">
```

Comment:
This doesn't seem to follow on correctly from the previous line. Did you mean to insert this further down?